airflow/providers/amazon/CHANGELOG.rst

```diff
@@ -26,6 +26,59 @@
 Changelog
 ---------
 
+8.8.0
+.....
+
+.. note::
+  This release of provider is only available for Airflow 2.5+ as explained in the
+  `Apache Airflow providers support policy <https://github.com/apache/airflow/blob/main/PROVIDERS.rst#minimum-supported-version-of-airflow-for-community-managed-providers>`_.
+
+Features
+~~~~~~~~
+
+* ``Implements 'AwsBaseOperator' and 'AwsBaseSensor' (#34784)``
+* ``Extend hooks arguments into 'AwsBaseWaiterTrigger' (#34884)``
+* ``Allow setup 'endpoint_url' per-service in AWS Connection (#34593)``
+
+Bug Fixes
+~~~~~~~~~
+
+* ``fix(providers/amazon): respect soft_fail argument when exception is raised (#34134)``
+* ``do not fail operator if we cannot find logs (#34570)``
+* ``Respect 'soft_fail' argument when running 'BatchSensors' (#34592)``
+* ``Respect 'soft_fail' argument when running 'SqsSensor' (#34569)``
+* ``Respect 'soft_fail' argument when running 'EcsBaseSensor' (#34596)``
+* ``Respect 'soft_fail' argument when running 'SageMakerBaseSensor' (#34565)``
+* ``Respect 'soft_fail' parameter in 'S3KeysUnchangedSensor' and 'S3KeySensor' (#34550)``
+* ``Respect 'soft_fail' parameter in 'LambdaFunctionStateSensor' (#34551)``
+* ``Respect 'soft_fail' parameter in 'AthenaSensor' (#34553)``
+* ``Respect 'soft_fail' parameter in 'QuickSightSensor' (#34555)``
+* ``Respect 'soft_fail' parameter in 'GlacierJobOperationSensor' (#34557)``
+* ``Respect 'soft_fail' parameter in 'GlueJobSensor', 'GlueCatalogPartitionSensor' and 'GlueCrawlerSensor' (#34559)``
+* ``Respect 'soft_fail' parameter in 'StepFunctionExecutionSensor' (#34560)``
+
+Misc
+~~~~
+
+* ``Refactor consolidate import from io in providers (#34378)``
+* ``Upgrade watchtower to 3.0.1 (#25019) (#34747)``
+* ``Bump min airflow version of providers (#34728)``
+* ``Refactor: consolidate import time in providers (#34402)``
+* ``Include AWS Lambda execution logs to task logs (#34692)``
```

Comment:
I assume that #34692 could be classified as a new a new feature. Previously we do not write execution logs (last 4 KB in sync mode) into the Airflow tasks logs

Reply:
will change